lib/app.js

```diff
@@ -1,12 +1,23 @@
 const express = require('express');
-
+const cookieParser = require('cookie-parser');
+const cors = require('cors');
 const app = express();
 
 // Built in middleware
 app.use(express.json());
-
+app.use(cookieParser());
+app.use(
+  cors({
+    origin: [
+      'https://leafy-snickerdoodle-bee210.netlify.app',
+      'http://localhost:5500',
+    ],
+    credentials: true,
+  })
+);
 // App routes
-
+app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/todos', require('./controllers/todos'));
```

Comment:
since all of the ToDos are using the `authenticate` middleware, you can define that here
`app.use('/api/v1/todos', authenticate, require('./controllers/todos'));`